Update replies documentation

john-westcott-iv opened the pull request:
<!--- changelog-entry
# Fill in 'msg' below to have an entry automatically added to the next release changelog.
# Leaving 'msg' blank will not generate a changelog entry for this PR.
# Please ensure this is a simple (and readable) one-line string.
---
msg: ""
-->

##### SUMMARY
<!--- Describe the change, including rationale and design decisions -->
Adding PR Denied
Creating headers for PRs, Mailing List and Common items

<!---
If you are fixing an existing issue, please include "related #nnn" in your
commit message and your description; but you should still explain what
the change does.
-->

##### ISSUE TYPE
<!--- Pick one below and delete the rest: -->
 - Docs Pull Request

##### COMPONENT NAME
<!--- Name of the module/plugin/module/task -->
 - Docs

##### AWX VERSION
<!--- Paste verbatim output from `make VERSION` between quotes below -->
```
awx: 21.0.1.dev154+gfbf25408df.d20220601
```


##### ADDITIONAL INFORMATION
<!---
Include additional information to help people understand the change here.
For bugs that don't have a linked bug report, a step-by-step reproduction
of the problem is helpful.
  -->

<!--- Paste verbatim command output below, e.g. before and after your change -->
```

```

john-westcott-iv committed:
Adding PR Denied and creating headers for PRs, Mailing List and Common items

john-westcott-iv committed:
Touching up the no thanks and adding PR timeout

samccann commented on .github/triage_replies.md:
```diff
@@ -3,37 +3,81 @@
+## PRs
+
+### Visit our mailing list
 - Hello, your question seems like a good one to ask on our mailing list at https://groups.google.com/g/awx-project. You can also join #ansible-awx on https://libera.chat/ and ask your question there.
```
This should be the matrix channel instead of IRC as the preferred chat option.

john-westcott-iv replied:
That would need to change in a lot of places and we should do a separate PR for that. In fact, what would you think about just changing this response to point to https://github.com/ansible/awx/#get-involved so we have one less reference to start with?

samccann replied:
yes pointing to one place where you can update it is a good idea.

rebeccahhh replied:
+1

john-westcott-iv replied:
@samccann Let me know what you think of this new wording here.

samccann commented on .github/triage_replies.md:
```diff
@@ -3,37 +3,81 @@
+\
+(A) INSERT ITEM HERE\
+\
+Please know that we are always up for discussion but this project is very active. Because of this, we're unlikely to see comments made on closed PRs and we lock them after some time. If you or anyone else has any further questions, please let us know by using any of the communication methods listed in the page below:\
```
```suggestion
Please know that we are always up for discussion but this project is very active. Because of this, we're unlikely to see comments made on closed PRs, and we lock them after some time. If you or anyone else has any further questions, please let us know by using any of the communication methods listed in the page below:\
```

samccann commented on .github/triage_replies.md:
```diff
@@ -3,37 +3,81 @@
+\
+Thank you very much for your submission to AWX. It means a lot to us that you have taken time to contribute.\
+\
+On this PR changes were requested but it has been some time since then. We think this PR has merit but without the requested changes we are unable to merge it. At this time we are closing you PR. If you get time to address the changes you are welcome to open another PR or we can reopen this PR upon request if you contact us by using any of the communication methods listed in the page below:\
```
```suggestion
On this PR, changes were requested but it has been some time since then. We think this PR has merit but without the requested changes we are unable to merge it. At this time we are closing you PR. If you get time to address the changes you are welcome to open another PR or we can reopen this PR upon request if you contact us by using any of the communication methods listed in the page below:\
```

samccann replied:
I don't know if AWX has the bot that can add 'needs_info' and give PR owners a heads up a month or so in advance of closing their PR due to no replies/followup, but that would be nice to add.

john-westcott-iv replied:
I don't think we have anything like that but we should. It would be nice if we could add an "inactive" label which would reply with this and then close in 1mo if no additional activity happens on the PR.

samccann replied:
the ansible-core team has a bot that does this. I don't know any of the details but I think this is the code - https://github.com/ansible/ansibullbot.  It does a LOT of handy things wrt pr and issue mgmt.

rebeccahhh replied:
@samccann we've discussed following core's example a couple times ^ I think it just needs enough steam to get setup, we're stretched a bit thin at the moment but we'll get there eventually. 🤞🏻

john-westcott-iv replied:
I think we used to have an 'awxbot' that was similar to ansibullbot but I'm not sure why we stopped using it.

samccann commented on .github/triage_replies.md:
```diff
@@ -3,37 +3,81 @@
 
-## Give us more info
+### Give us more info
 - Hello, we'd love to help but we need a little more information about the problem you're having. Screenshots, log outputs, or any reproducers would be very helpful. 
```
```suggestion
- Hello, we'd love to help, but we need a little more information about the problem you're having. Screenshots, log outputs, or any reproducers would be very helpful. 
```

samccann commented on .github/triage_replies.md:
```diff
@@ -3,37 +3,81 @@
 
-## Receptor
+### Code of Conduct
+- Hello. Please keep in mind that Ansible adheres to a Code of Conduct in its community spaces. The spirit of the code of conduct is to be kind, and this is your friendly reminder to be so. Please see the full code of conduct here if you have questions: https://docs.ansible.com/ansible/latest/community/code_of_conduct.html 
```
```suggestion
- Hello. Please keep in mind that Ansible adheres to a Code of Conduct in its community spaces. The spirit of the code of conduct is to be kind. Please see the full code of conduct here if you have questions: https://docs.ansible.com/ansible/latest/community/code_of_conduct.html 
```

samccann replied:
unless this auto-reply is a warning for a CoC infringement, I'd suggest removing 'this is your friendly reminder' bit as it sounds... ominous to me.

john-westcott-iv replied:
I think was intended to be a warning for CoC. @rebeccahhh, is that correct?

rebeccahhh replied:
this is indeed a warning for CoC violations.

john-westcott-iv replied:
Resolving since this is a CoC warning.

samccann commented on .github/triage_replies.md:
```diff
@@ -3,37 +3,81 @@
+## Mailing List Triage
+
+### Create an issue
+- Hello, thanks for reaching out on list. We think this merits an issue on our Github, https://github.com/ansible/awx/issues. If you could open an issue up on Github it will get tagged and integrated into our planning and workflow. All future work will be tracked there.
```
```suggestion
- Hello, thanks for reaching out on list. We think this merits an issue on our Github, https://github.com/ansible/awx/issues. If you could open an issue up on Github it will get tagged and integrated into our planning and workflow. All future work will be tracked there. Issues should include as much information as possible, including screenshots, log outputs, or any reproducers.
```

samccann commented on .github/triage_replies.md:
```diff
@@ -3,37 +3,81 @@
+- Hello, thanks for reaching out on list. We think this merits an issue on our Github, https://github.com/ansible/awx/issues. If you could open an issue up on Github it will get tagged and integrated into our planning and workflow. All future work will be tracked there.
+
+### Create a Pull Request
+- Hello, we think your idea is good, please consider contributing a PR for this, following our contributing guidelines: https://github.com/ansible/awx/blob/devel/CONTRIBUTING.md
```
```suggestion
- Hello, we think your idea is good! Please consider contributing a PR for this following our contributing guidelines: https://github.com/ansible/awx/blob/devel/CONTRIBUTING.md
```

samccann commented on .github/triage_replies.md:
```diff
@@ -3,37 +3,81 @@
+
+### Receptor
 - You can find the receptor docs here: https://receptor.readthedocs.io/en/latest/ 
 - Hello, your issue seems related to receptor, could you please open an issue in the receptor repository? https://github.com/ansible/receptor. Thanks!
```
```suggestion
- Hello, your issue seems related to receptor. Could you please open an issue in the receptor repository? https://github.com/ansible/receptor. Thanks!
```

samccann commented on .github/triage_replies.md:
```diff
@@ -3,37 +3,81 @@
 
-## Ansible Galaxy not AWX
+### Ansible Galaxy not AWX
 - Hey there, that sounds like an FAQ question, did this: https://www.ansible.com/products/awx-project/faq cover your question?
```
```suggestion
- Hey there. That sounds like an FAQ question. Did this: https://www.ansible.com/products/awx-project/faq cover your question?
```

samccann reviewed (commented):
sprinkled some docs nits as suggested edits.

john-westcott-iv committed:
Making grammer and minor changes

john-westcott-iv committed:
Removing reference to mailing list and IRC in favor of README

Adding clearer templates for version numbers in AWX Release message

Inviting feedback in Try latest version

samccann reviewed (approved):
LGTM!
